winget-install.ps1

```diff
@@ -156,24 +156,40 @@ function Add-ScopeMachine {
 }
 
 #Check if app is installed
-function Confirm-Install ($AppID) {
-    #Get "Winget List AppID"
-    $InstalledApp = & $winget list --Id $AppID -e --accept-source-agreements | Out-String
+function Confirm-Install ($AppID, $AppArgs) {
+    #Get "Winget List AppID" and remove any header information
+    $InstalledApp = & $winget list --Id $AppID -e --accept-source-agreements
+    $InstalledApp = $InstalledApp[$InstalledApp.Length-1]
+    [String]$InstalledVersion = ($InstalledApp.Split($AppID)[1]).Trim().Split(" ")[0]

```

Comment:
The thing is, we are not 100% sure that the `winget list` output will remain the same over time. Sometimes they add extra lines on certains commands and stuff :/

Reply:
Hm, yes, you are right, but i don't see another option here to implement this properly.
If this project keeps relying on the default winget commands and not switch to using the powershell cmdlet, then this could always happen, though imo this is a problem, that might happen or might not, we dont know and this shouldnt stop new features from getting implemented.

Reply:
we can maybe use the `winget export -o temp.json -s winget --include-versions` command to compare app and version on json format.

Reply:
Hey @Romanitho i had some time to look at it and i think i have solved this now and it should work great. The only "problem" is the winget export command can be slow, atleast slower than the winget list command, but i think using the json output its more resilient to changes this way.

Reply:
it takes only few seconds, so for me json format is much more reliable than shell output